js/components/controlBar.js

```diff
@@ -119,11 +125,23 @@ var ControlBar = React.createClass({
   closeQualityPopover: function() {
     if(this.props.controller.state.videoQualityOptions.showVideoQualityPopover == true) {
       this.toggleQualityPopover();
+      $("span.oo-icon.oo-icon-bitrate").css("color", this.props.skinConfig.controlBar.iconStyle.inactive.color);
```

Comment:
@mistermendez I am not sure how we can achieve this without jQuery as icon element is not accessible in this method. Do you have any suggestion?